tests/inference/test_example_query_openai_sdk.py

```diff
@@ -57,10 +64,15 @@ def script_with_args(api_base, model_name, streaming_response, max_new_tokens, t
 
     result_openai = subprocess.run(cmd_openai, capture_output=True, text=True)
 
+    # Ensure there are no errors in the OpenAI API query script execution
+    try:
+        assert result_openai.returncode == 0, print(result_serve.stderr)
```

Comment:
this seems strange to me to use ` ,` to seperate two statements.

Reply:
It seems that the official usage of assert is to execute the content after “，” after the assertion fails

Reply:
yes, but it is assert x > y, "x is not greater than y", not print(...)